Makefile

```diff
@@ -98,11 +98,11 @@ lint-dependencies:
 	@$(PYTHON_PIP) install -q -r lint_requirements.txt
 
 lint-server: lint-dependencies
-	flake8 elyra
-	black --check --diff --color . || (echo "Black formatting encountered issues.  Use 'make black-format' to apply the suggested changes."; exit 1)
+	$(PYTHON) -m flake8 elyra
+	@$(PYTHON) -m black --check --diff --color . || (echo "Black formatting encountered issues.  Use 'make black-format' to apply the suggested changes."; exit 1)
```

Comment:
Since this no longer echos the `black` command, yet does echo the `flake8` command (which I think is helpful), we should also echo the black command.  Unfortunately, due to the error handling required to echo instructions on failure, this was the only way I could get both items to work.  The variable is in place simply to avoid redundancy, but not necessary otherwise (sorry, suggestions aren't supported across deleted items and the suggestion requires changes prior to the target)...
```Makefile
BLACK_CMD:=$(PYTHON) -m black --check --diff --color .
lint-server: lint-dependencies
        $(PYTHON) -m flake8 elyra
        @echo $(BLACK_CMD)
        @$(BLACK_CMD) || (echo "Black formatting encountered issues.  Use 'make black-format' to apply the suggested changes."; exit 1)
```
On failures, this results in...
```
$ make lint-server
python3 -m flake8 elyra
python3 -m black --check --diff --color .
Skipping .ipynb files as Jupyter dependencies are not installed.
You can fix this by running ``pip install black[jupyter]``
--- elyra/metadata/metadata.py	2022-04-14 22:33:11.888517 +0000
+++ elyra/metadata/metadata.py	2022-04-15 18:09:02.398194 +0000
@@ -35,11 +35,11 @@
     schema_name = None
     metadata = {}
     reason = None
 
     def __init__(self, **kwargs: Any) -> None:
-        self.name = kwargs.get('name')
+        self.name = kwargs.get("name")
         self.display_name = kwargs.get("display_name")
         self.schema_name = kwargs.get("schema_name")
         self.metadata = kwargs.get("metadata", {})
         self.resource = kwargs.get("resource")
         self.reason = kwargs.get("reason")
would reformat elyra/metadata/metadata.py
Oh no! 💥 💔 💥
1 file would be reformatted, 130 files would be left unchanged.
Black formatting encountered issues.  Use 'make black-format' to apply the suggested changes.
make: *** [lint-server] Error 1
```
And on success...
```
$ make lint-server
python3 -m flake8 elyra
python3 -m black --check --diff --color .
Skipping .ipynb files as Jupyter dependencies are not installed.
You can fix this by running ``pip install black[jupyter]``
All done! ✨ 🍰 ✨
131 files would be left unchanged.
```